app/connectors/httpParsers/PaymentsHistoryHttpParser.scala

```diff
@@ -50,7 +50,8 @@ object PaymentsHistoryHttpParser extends ResponseHttpParsers {
       FinancialTransactionsConstants.vatDefaultSurcharge,
       FinancialTransactionsConstants.errorCorrectionCreditCharge,
       FinancialTransactionsConstants.errorCorrectionDebitCharge,
-      FinancialTransactionsConstants.vatRepaymentSupplement
+      FinancialTransactionsConstants.vatRepaymentSupplement,
+      FinancialTransactionsConstants.vatAdditionalAssessmentInterest
```

Comment:
I don't think we are doing the chargeTypes for the payment history as a part of this story, just the 'what you owe' page stuff, so I think this should be done in the PaymentHttpParser